contracts/upgradeable_contracts/erc20_to_native/ForeignBridgeErcToNative.sol

```diff
@@ -171,7 +175,61 @@ contract ForeignBridgeErcToNative is BasicForeignBridge, ERC20Bridge, OtherSideB
         emit TokensSwapped(hdToken, fdToken, curHDTokenBalance);
     }
 
-    function relayTokens(address _from, address _receiver, uint256 _amount, address _token) external {
+    function setMinDaiTokenBalance(uint256 _minBalance) external onlyOwner {
```

Comment:
If all these new methods are valid only in case of Chai used by the bridge does it makes sense to move them to the `ChaiConnector` module?